packages/mermaid/src/diagrams/sankey/styles.js

```diff
@@ -0,0 +1,5 @@
+const getStyles = (options) =>
+  `
+`;
+
+export default getStyles;
```

Comment:
You don't need this file if it's empty. 
You can directly put the function `() => {}` instead.

Reply:
Done

Reply:
@Yokozuna59 didn't we make styles optional? 

I doubt that's merged yet, though. Maybe @nirname could cherry-pick that commit.

Reply:
> @Yokozuna59 didn't we make styles optional?
> 
> I doubt that's merged yet, though.

Yes, it's been optional when we have updated the info graph.

https://github.com/mermaid-js/mermaid/blob/3d117815425cbd05105ec0f0192adc0be91931f2/packages/mermaid/src/diagram-api/types.ts#L31

Reply:
There is no such a file in my project right now, it's a bug in GH, perhaps